### PR TITLE
Fix bad merge of security patch from 17.05 into dev.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by dannon** [comment]
Argh, good catch.  My fault not noticing this in the original :/

I don't even see the duplicated method in the original merge PR, it's weird how this stuff shows up on GH sometimes :(

**Comment by martenson** [comment]
@dannon I have been wrapping my head around this for a bit, I had no conflict of this type when merging (just parens whitespace). I assume affected code had to change significantly in dev since 17.05.

**Comment by dannon** [comment]
Yeah, it's weird, slipped past both of us @martenson :/

**Comment by martenson** [comment]
As a bonus GitHub now displays two security patch commits in dev
screenshot 2017-08-28 17 03 16

**Comment by dannon** [comment]
Double patched, extra secure.

**Comment by hexylena** [comment]
Oh jeez, so sorry about this @jmchilton. Thanks for patching it.

File: lib/galaxy/datatypes/data.py [hunk 1]
```diff
@@ -372,7 +372,7 @@ def display_data(self, trans, data, preview=False, filename=None, to_ext=None, *
                     tmp_file_name = tmp_fh.name
```
Comment (martenson) [inline lib/galaxy/datatypes/data.py:372]: this variable is unused

**Pull request merged**